File: university_projects/car_rental/cpp_version/Offer.cpp
#include "Offer.h"

Offer::Offer(int id, float price, Car^ car) :offerId(id), offerPrice(price), car(car)
{
}


int Offer::getOfferId()
{
	return this->offerId;
}


float Offer::getOfferPrice()
{
	return this->offerPrice;
}


String^ Offer::getOfferCarBrand()
{
	return this->car->getCarInfoBrand();
}

String^ Offer::getOfferCarModel()
{
	return this->car->getCarInfoModel();
}

int Offer::getOfferCarId()
{
	return this->car->getCarId();
}

File: university_projects/car_rental/android_version/java/com/example/wiktorpieklik/car_rental/Model/Offer.java
package com.example.wiktorpieklik.car_rental.Model;

public class Offer
{
    private int offerId;
    private float offerPrice;
    private Car car;

    public Offer(int id, float price, Car car)
    {
        offerId=id;
        offerPrice=price;
        this.car=car;
    }

    public int getOfferId()
    {
        return this.offerId;
    }

    public float getOfferPrice()
    {
        return this.offerPrice;
    }

    String getOfferCarBrand()
    {
        return this.car.getCarInfoBrand();
    }

    String getOfferCarModel()
    {
        return this.car.getCarInfoModel();
    }

    public int getOfferCarId()
    {
        return this.car.getCarId();
    }
}
